client/app/dbaas/logs/detail/roles/logs-roles.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Roles
```

Comment:
hardcoded values. Will be translated later I guess?

Reply:
This change got into the PR because of merge conflicts. It is not a part of the PR now :). Will be fixed when we work on this.